package.json

```diff
@@ -26,6 +26,7 @@
     "react-dom": "^17.0.2",
     "react-error-boundary": "^3.1.4",
     "react-markdown": "^8.0.3",
+    "react-ga4": "^1.4.1",
```

Comment:
Why is this  `1.4.1` and the other `package.json` has it as `1.4.0`. This was not needed here before.